projects/plugins/jetpack/extensions/shared/external-media/editor.scss

```
@@ -447,3 +447,11 @@ $grid-size: 8px;
 .editor-post-featured-image .components-dropdown {
 	display: initial;
 }
+
+.block-editor-inserter__media-panel .components-search-control input[type=search].components-search-control__input[placeholder~="Google"] {
+	display: none;
+}
+
+.block-editor-inserter__media-panel .components-search-control input[type=search].components-search-control__input[placeholder~="Google"] + .components-search-control__icon {
+	display: none;
+}
```

Nitpick (non-blocking): Since this is a SCSS file, we can simplify a little bit the syntax. It'd be great to also have a newline at the end of the file.
```suggestion
.block-editor-inserter__media-panel .components-search-control input[type=search].components-search-control__input[placeholder~="Google"] {
	display: none;
	& + .components-search-control__icon {
		display: none;
	}
}

```